roblox/jobs.py

```diff
@@ -144,3 +146,123 @@ def __init__(self, client: Client, data: dict):
             ) for instance_data in data["Collection"]
         ]
         self.total_collection_size: int = data["TotalCollectionSize"]
+
+
+class ServerType(Enum):
+    """
+    Represents the type of server.
+    """
+    
+    Public = 0
+    Friend = 1
```

Comment:
Roblox accepts strings for this parameter instead of numbers (their docs are wrong) so go with that instead. also the casing should match other enums